variables.sqlserver.tf

```diff
@@ -0,0 +1,64 @@
+variable "server_version" {
+  type        = string
+  description = "(Required) The version for the new server. Valid values are: 2.0 (for v11 server) and 12.0 (for v12 server). Changing this forces a new resource to be created."
+  nullable    = false
+}
+
+variable "administrator_login" {
+  type        = string
+  default     = null
+  description = "(Optional) The administrator login name for the new server. Required unless `azuread_authentication_only` in the `azuread_administrator` block is `true`. When omitted, Azure will generate a default username which cannot be subsequently changed. Changing this forces a new resource to be created."
+}
+
+variable "administrator_login_password" {
+  type        = string
+  default     = null
+  description = "(Optional) The password associated with the `administrator_login` user. Needs to comply with Azure's [Password Policy](https://msdn.microsoft.com/library/ms161959.aspx). Required unless `azuread_authentication_only` in the `azuread_administrator` block is `true`."
+  sensitive   = true
+}
+
+variable "connection_policy" {
+  type        = string
+  default     = null
+  description = "(Optional) The connection policy the server will use. Possible values are `Default`, `Proxy`, and `Redirect`. Defaults to `Default`."
+}
+
+variable "azuread_administrator" {
+  type = object({
+    azuread_authentication_only = optional(bool)
```

Comment:
From a WAF perspective, should we default this to true?

Reply:
Will consider post release